x-pack/solutions/security/plugins/security_solution/public/detections/components/alerts_table/translations.ts

```diff
@@ -418,3 +418,10 @@ export const EVENT_RENDERED_VIEW_COLUMNS = {
     defaultMessage: 'Event Summary',
   }),
 };
+
+export const NOT_ENOUGH_PRIVILEGES = i18n.translate(
+  'xpack.securitySolution.detectionEngine.alerts.notEnoughPrivileges',
+  {
+    defaultMessage: 'Not enough privileges',
```

Comment:
Changes look good. Thanks for fixing this. I would also consult @natasha-moore-elastic for confirmation on this text or any alternatives.